Convert impersonal "users" to direct "you" in documentation (#4959)
ClickHouse/clickhouse-docs: docs/use-cases/observability/clickstack/migration/elastic/intro.md

## Migrating to ClickStack from Elastic {#migrating-to-clickstack-from-elastic}

> This guide is intended if you're migrating from the Elastic Stack — specifically if you're using Kibana to monitor logs, traces, and metrics collected via Elastic Agent and stored in Elasticsearch. It outlines equivalent concepts and data types in ClickStack, explains how to translate Kibana Lucene-based queries to HyperDX's syntax, and provides guidance on migrating both data and agents for a smooth transition.

Reviewer: ```suggestion
This guide is intended for users migrating from the Elastic Stack — specifically those using Kibana to monitor logs, traces, and metrics collected via Elastic Agent and stored in Elasticsearch. It outlines equivalent concepts and data types in ClickStack, explains how to translate Kibana Lucene-based queries to HyperDX's syntax, and provides guidance on migrating both data and agents for a smooth transition.
```
